crafts.lua

```diff
@@ -5,35 +5,68 @@
 --    UNDER TERMS OF WTFPL LICENSE
 
 -- 2012, 2013, 2014 obneq aka jin xi
+if minetest.get_modpath("mcl_core") then
```

Comment:
Please add mcl_core as optional dependency to this mod so that this code works reliably.